pkg/vulnsrc/vulnerability/const.go

```diff
@@ -70,3 +72,36 @@ var Ecosystems = []types.Ecosystem{
 	Bitnami,
 	Kubernetes,
 }
+
+// AllSourceIDs is a list of all supported sourceIDs.
+// This order affects the choice of `Severity`, `Title`, `Description` and `CweIDs`.
+// e.g. if `NVD` has no severity - we will keep the severity of `RedHat` (if it exists), etc.
```

Comment:
nit

```suggestion
// AllSourceIDs lists all supported vulnerability source IDs in order of precedence.
// When searching for vulnerability details (Severity, Title, Description, and CWE-IDs),
// the sources are checked in this order until valid data is found.
// For example, if severity data is missing in NVD, it will check Red Hat next,
// continuing through the list until it finds a valid severity value.```